src/main/proto/user.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package user;
+
+import "google/protobuf/empty.proto";
+
+option java_package = "com.godeltech.grpc.user";
+option java_multiple_files = true;
+
+message UserDto {
+  uint64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string email = 4;
+  string phone_number = 5;
+}
+
+message UserIdDto {
+  uint64 id = 1;
```

Comment:
I'd suggest rename `UserIdDto` to `GetUserRequest` like as `CreateUserRequest` etc